Translated files Dutch language

Alex-e107nl opened the pull request:
Dutch translation as currently used on e107.nl. Thank you Moc!!

Alex-e107nl committed:
Translated files Dutch language

Dutch translation as currently used on e107.nl. Thank you Moc!!

Moc commented on languages/Dutch/Dutch_front.php:
```diff
@@ -0,0 +1,33 @@
+
+
+// Verify
+define("LAN_2FA_VERIFY_INSTRUCTIONS", 		"Open de Twee Factor Authenticatie-app op uw apparaat om uw authenticatiecode te bekijken en uw identiteit te verifi&#235;ren."); 
```
@Alex-e107nl This should be `verifiëren` rather than ` verifi&#235;ren`.  :)

Moc commented:
@Alex-e107nl Just some minor fixes to be made, otherwise perfect! Please see my comments above. If you can fix it, that would be great, otherwise I'll do it after merging.

Moc commented on languages/Dutch/Dutch_front.php:
```diff
@@ -0,0 +1,33 @@
+define("LAN_2FA_DISABLE_INSTRUCTIONS", 	"Om tweefactorauthenticatie voor je account [b]uit te schakelen[/b], voer je ter verificatie je zescijferige code in.");
+
+define("LAN_2FA_ENABLED", 				"Tweefactorauthenticatie is met succes [b]ingeschakeld[/b]"); 
+define("LAN_2FA_DISABLED", 				"Tweefactorauthenticatie is succesvol [uitgeschakeld[/b]"); 
```
Check the [b] on "uitgeschakeld"

Moc commented on languages/Dutch/Dutch_front.php:
```diff
@@ -0,0 +1,33 @@
+ */
+
+// Page titles
+define("LAN_2FA_TITLE", "Two Factor Authentication"); 
```
This one can be translated as well :)

Moc commented on languages/Dutch/Dutch_front.php:
```diff
@@ -0,0 +1,33 @@
+
+// Setup
+define("LAN_2FA_ENABLE_INSTRUCTIONS1", 	"Om twee-factorenauthenticatie voor je account in te stellen, gebruikt je de [authenticator-app] op je apparaat om de onderstaande QR-code te scannen of om de geheime sleutel handmatig in te voeren."); // Do not remove the brackets [...] - they are used for links
+define("LAN_2FA_ENABLE_INSTRUCTIONS2", "Om te bevestigen dat uw authenticatie-app de juiste 6-cijferige code retourneert, verifieert u de code door deze hieronder in te voeren. Na verificatie wordt twee-factor-authenticatie ingesteld voor uw account.");
```
"Je" or "u"? You're using both, for consistency I would change this line to "je", so that you use "je" everywhere.

Alex-e107nl commented:
I would like to correct errors, but I don't see where you indicated which errors they are? I just search and search... tokens and emoticons that disappear on e107.nl and now here....

Moc commented:
If you check this page in your browser, you will see my comments: https://github.com/e107inc/twofactorauth/pull/7

Alex-e107nl commented:
Hihihi not really... i just get back on this page... #7 link does nothing for me... maybe you can do it please :-)

Moc commented:
@Alex-e107nl Sorry, that was my mistake. Now it should be visible :)

Alex-e107nl committed:
Bugs corrected

Next time it will be better, you are Dutch so you see it... I wanted to be too fast, so much to do...

Alex-e107nl commented:
I think I have now sent the correct files, which is quite difficult for the first time.

Moc commented:
But you did it succesfully! Great job!
